tests/unit_tests/service/test_interface.py

```diff
@@ -13,6 +13,7 @@
 )
 from ophyd.sim import SynAxis
 from stomp.connect import StompConnection11 as Connection
+from tests.unit_tests.service.test_rest_api import FAKE_INSTRUMENT_SESSION
```

Comment:
Test modules depending on other test modules is a bit weird. Should this be a fixture or a per module constant? The `test_client` module is using literals everywhere.